nservicebus/messaging/delayed-delivery.md

```diff
@@ -25,6 +36,13 @@ snippet:delayed-delivery-datetime
 
 In order to use message deferral the Timeout Manager feature must be enabled. When you defer a message, it is sent to the timeout manager requesting it to deliver the message at a later time. 
 
+snippet:configure-timeout-manager
+
+NOTE: In Version 4 Timeout Manager is enabled by default.
```

Comment:
Yeah I verified it and you are right, I will remove the snippets and add approperiate comments.